# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1517600838 18000
#      Fri Feb 02 14:47:18 2018 -0500
# Node ID 1c1789fa5d89809eaf5ac725129f63d4aaf77ea7
# Parent  4f8edc1b0f1852211ffa0ccd074e0883f633550d
remove accidental change in variable editor model

* variab-editor-model.cc (variable_editor_model::flags):
Restore code commented out for debugging.  Update coding style.

diff --git a/libgui/src/variable-editor-model.cc b/libgui/src/variable-editor-model.cc
--- a/libgui/src/variable-editor-model.cc
+++ b/libgui/src/variable-editor-model.cc
@@ -684,29 +684,25 @@ variable_editor_model::setData (const QM
     (this, &variable_editor_model::set_data_oct, r, c, vstr.toStdString ());
 
   return true;
 }
 
 Qt::ItemFlags
 variable_editor_model::flags (const QModelIndex& idx) const
 {
-  if (m_d->m_validity)
-    {
-#if 0
-      if (requires_sub_editor (idx))
-        {
-          if (editor_type (idx) != sub_string)
-            return QAbstractTableModel::flags (idx);
-        }
-#endif
-      return QAbstractTableModel::flags (idx) | Qt::ItemIsEditable;
-    }
+  if (! m_d->m_validity)
+    return Qt::NoItemFlags;
+
+  Qt::ItemFlags retval = QAbstractTableModel::flags (idx);
 
-  return Qt::NoItemFlags;
+  if (! requires_sub_editor (idx))
+    retval |= Qt::ItemIsEditable;
+
+  return retval;
 }
 
 bool
 variable_editor_model::insertRows (int row, int count, const QModelIndex&)
 {
   // FIXME: cells?
 
   octave_link::post_event <variable_editor_model, std::string, std::string>
